### PR TITLE
Jenkins matrix improvement to support Pandas 1.2.0 and 1.3.0

### PR DESCRIPTION
**Pull request by V1NAY8**
To-do
- [x] Jenkins Matrix
- [ ] Github Actions Matrix

### CODE REVIEW

**Comment by sethmlarson** [comment]
@V1NAY8 Lets do Github Actions separate? Gives a chance to fix any problems with Jenkins too

File: .ci/run-repository.sh [hunk 1]
```diff
@@ -21,6 +21,7 @@ echo -e "\033[34;1mINFO:\033[0m VERSION ${ELASTICSEARCH_VERSION}\033[0m"
 echo -e "\033[34;1mINFO:\033[0m CONTAINER ${ELASTICSEARCH_CONTAINER}\033[0m"
 echo -e "\033[34;1mINFO:\033[0m TEST_SUITE ${TEST_SUITE}\033[0m"
 echo -e "\033[34;1mINFO:\033[0m PYTHON_VERSION ${PYTHON_VERSION}\033[0m"
+echo -e "\033[34;1mINFO:\033[0m PYTHON_VERSION ${PANDAS_VERSION}\033[0m"
```
Comment (sethmlarson) [inline .ci/run-repository.sh:24]: PANDAS_VERSION instead of PYTHON_VERSION

**Review by sethmlarson — APPROVED** [review]
LGTM, one fix before merge

**Comment by V1NAY8** [comment]
All good, merge it once.

**Review by sethmlarson — APPROVED** [review]

**Pull request merged**